Store/pages/account-address-edit.xml

```diff
@@ -18,27 +18,21 @@
 				<property name="size" type="integer">35</property>
 			</widget>
 		</widget>
-		<widget class="SwatFormField" id="country_field">
-			<property name="title" translatable="yes">Location</property>
-			<widget class="SwatFlydown" id="country">
-				<property name="required" type="boolean">true</property>
-				<property name="show_blank" type="boolean">true</property>
-			</widget>
-		</widget>
 		<widget class="SwatFormField" id="address_line1_field">
 			<property name="title" translatable="yes">Street Address</property>
-			<widget class="SwatTextarea" id="line1">
+			<widget class="SwatEntry" id="address_line1">
```

Comment:
I __think__ this screws up address saving, `StoreAccountAddressEditPage` is expecting this to be `line1`.